ktx

```diff
@@ -34,7 +34,7 @@ ktx() {
     fi
 
     # If argument is "None" then unset the KUBECONFIG variable
-    if [ "$1" == "none" ] || [ "$1" == "None" ]; then
+    if [ "$1" = "none" ] || [ "$1" = "None" ]; then
```

Comment:
```suggestion
    if [[ "$1" == "none" ]] || [[ "$1" == "None" ]]; then
```
This will work across zsh and bash. Thanks to @asenchi and @awwithro for helping out on this one!